src/test/java/uk/gov/hmcts/darts/event/service/impl/CaseManagementRetentionServiceImplTest.java

```diff
@@ -0,0 +1,126 @@
+package uk.gov.hmcts.darts.event.service.impl;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.darts.common.entity.CaseManagementRetentionEntity;
+import uk.gov.hmcts.darts.common.entity.CourtCaseEntity;
+import uk.gov.hmcts.darts.common.entity.CourthouseEntity;
+import uk.gov.hmcts.darts.common.entity.EventEntity;
+import uk.gov.hmcts.darts.common.entity.RetentionPolicyTypeEntity;
+import uk.gov.hmcts.darts.common.exception.DartsApiException;
+import uk.gov.hmcts.darts.common.helper.CurrentTimeHelper;
+import uk.gov.hmcts.darts.common.repository.CaseManagementRetentionRepository;
+import uk.gov.hmcts.darts.common.repository.RetentionPolicyTypeRepository;
+import uk.gov.hmcts.darts.common.util.CommonTestDataUtil;
+import uk.gov.hmcts.darts.event.exception.EventError;
+import uk.gov.hmcts.darts.event.model.DartsEventRetentionPolicy;
+
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.darts.event.service.impl.CourtLogsServiceImplTest.CASE_0000001;
+
+
+@ExtendWith(MockitoExtension.class)
+class CaseManagementRetentionServiceImplTest {
+
+    static final String SWANSEA = "SWANSEA";
+
+    static final String ERROR_MESSAGE = """
+    Data on the event could not be reconciled with Darts records. Could not find a retention policy for fixedPolicyKey '1000'""";
+
+    @InjectMocks
```

Comment:
I think you've just shifted this class, but @InjectMocks is not needed here as its being explicitly instantiated.